src/main/java/util/Storage.java

```diff
@@ -0,0 +1,68 @@
+package util;
+
+import Tasks.Task;
+import Tasks.TaskList;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Storage is a class that deals with the management of the database
+ */
+public class Storage {
+    private String filePath;
+
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    public ArrayList<Task> load() throws FileNotFoundException {
+        ArrayList<Task> arr = new ArrayList<>();
```

Comment:
I think a better naming convention should be implemented, try to name `arr` to exactly what it stores, something like `storedTasks` or `tasksFromDb`.